universal-application-tool-0.0.1/app/views/LoginForm.java

```diff
@@ -54,17 +65,27 @@ public Content render(Http.Request request, Messages messages, Optional<String>
   private ContainerTag mainContent(Messages messages) {
     ContainerTag content = div().withClasses(BaseStyles.LOGIN_PAGE);
 
-    content.with(
-        this.layout
-            .viewUtils
-            .makeLocalImageTag("ChiefSeattle_Blue")
-            .withAlt("City of Seattle Logo")
-            .attr("aria-hidden", "true")
-            .withClasses(Styles.W_1_4, Styles.PT_4));
+    if (maybeLogoUrl.isPresent()) {
+      content.with(
+          img()
+              .withSrc(maybeLogoUrl.get())
+              .withAlt(civicEntityFullName + "logo")
```

Comment:
Nit: logo isn't capitalized here but is capitalized below.

Reply:
Done